samples/GraphQL.DataLoader.Example/DataLoaderGql.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
This file needs formatting.  Tabs are not uniform.

Reply:
Should be good now